pmm-tests/pmm-2-0-bats-tests/mongodb-tests.bats

```diff
@@ -255,6 +255,46 @@ skip "Skipping this test, because of setup issue on Framework, https://jira.perc
 	done
 }
 
-function teardown() {
+@test "PMM-T964 run pmm-admin add mongodb with --agent-password flag" {
+    COUNTER=0
+    IFS=$'\n'
```

Comment:
What do we have this for? Do we use it?

Reply:
it just breaks the output of pmm-admin list, helps to go through each entry in loop.